packages/subagents/src/agents/DevAgent.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  AgentOutputType,
+  Chat,
+  ChatMessageBuilder,
+  Env,
+  LlmApi,
+  Logger,
+  Scripts,
+  Workspace,
+  WrapClient,
+  agentPlugin,
+  trimText,
+} from "@evo-ninja/agent-utils";
+import { AgentConfig, SubAgent } from "../SubAgent";
+
+export interface DevAgentRunArgs {
+  goal: string;
+}
+
+const DEV_AGENT_CONFIG: AgentConfig<DevAgentRunArgs> = {
```

Comment:
If you're not exporting this, I think AGENT_CONFIG is better. Cuts down on the things to notice or change when copy pasting agents